src/app/inventory/ConnectedInventoryItem.tsx

```diff
@@ -16,6 +16,7 @@ interface ProvidedProps {
   id?: string; // defaults to item.index - id is typically used for `itemPop`
   allowFilter?: boolean;
   ignoreSelectedPerks?: boolean;
+  ignoreSelectedMods?: boolean;
```

Comment:
I was thinking of replacing both of these with a "showSelectedSubclass" property because that's all it's really used for.

Reply:
Yep, I'm not opposed to that. The complexity is that there are cases where we do want to show selected super for V3 subclasses but not for V2 subclasses (because we DIM can change the Super on V3 subclasses but not V2 subclasses). 

Maybe I should collapse those 2 booleans into a single enum value since there are only 3 valid options? e.g.
`selectedSuperDisplay: 'enabled' | 'disabled' | 'v3SubclassesOnly'`